pkg/sql/delegate/show_enums.go

```diff
@@ -15,24 +15,26 @@ import "github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
 func (d *delegator) delegateShowEnums() (tree.Statement, error) {
 	query := `
 SELECT
-	schema, name, string_agg(label, '|') AS value
+	schema, name, string_agg(label, '|'), owner AS value
```

Comment:
i think this should be:
`	schema, name, string_agg(label,) as value, owner`

Reply:
Fix in #55139 !

Reply:
Yeah, good catch.  Not sure how I missed that :-(

Reply:
easy to miss if logic test aren't printing out the column names :P